gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/RemoteGcsHelper.java

```diff
@@ -145,22 +145,17 @@ public static RemoteGcsHelper create(String projectId, String keyPath)
         log.log(Level.WARNING, ex.getMessage());
       }
       throw GcsHelperException.translate(ex);
-    } catch (IOException ex) {
-      if (log.isLoggable(Level.WARNING)) {
-        log.log(Level.WARNING, ex.getMessage());
-      }
-      throw GcsHelperException.translate(ex);
     }
   }
 
   /**
    * Creates a {@code RemoteGcsHelper} object. Project id and path to JSON key are read from two
-   * environment variables: {@code GCLOUD_TESTS_PROJECT_ID} and {@code GCLOUD_TESTS_KEY}.
+   * environment variables: {@code GCLOUD_PROJECT} and {@code GOOGLE_APPLICATION_CREDENTIALS}.
    *
    * @return A {@code RemoteGcsHelper} object for the provided options.
    * @throws com.google.gcloud.storage.testing.RemoteGcsHelper.GcsHelperException if environment
-   *     variables {@code GCLOUD_TESTS_PROJECT_ID} and {@code GCLOUD_TESTS_KEY} are not set or if
-   *     the file pointed by {@code GCLOUD_TESTS_KEY} does not exist
+   *     variables {@code GCLOUD_PROJECT} and {@code GOOGLE_APPLICATION_CREDENTIALS} are not set or
+   *     if the file pointed by {@code GOOGLE_APPLICATION_CREDENTIALS} does not exist
    */
   public static RemoteGcsHelper create() throws GcsHelperException {
```

Comment:
Explicit projetId and key verification is fine.

Another option would be to only document that this `create` method is based on the default lookup of [projectId](https://github.com/GoogleCloudPlatform/gcloud-java#specifying-a-project-id) and [credentials] (https://github.com/GoogleCloudPlatform/gcloud-java#authentication) which allows more more flexibility.

Is the current way your preference?